### ISSUE TITLE
canvas -> struct instead of class?

### ISSUE THREAD

**Issue by jamesmontemagno** [jonathanpeppers/lols#1]
Could this improve performance?

**Comment by jonathanpeppers** [comment]
I tried it, but I have a vague memory of getting about the same number. Let me try it again as maybe I could get a couple thousand more LOLs. LOL!

**Comment by jamesmontemagno** [comment]
Would love the see the canvas test in the main branch as well as I think that is a nice side by side ;)

**Comment by jonathanpeppers** [comment]
Yeah the canvas test is pretty sketchy. It's timing how fast it can use `Random`, create a C# object, and push in a `ConcurrentQueue`?

I'm not sure it's actually measuring anything meaningful.
